src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -149,4 +143,72 @@ boolean restoreDatabaseFromBackupFile(String storageLocation, String backupFileN
         }
         return false;
     }
+
+    /**
+     * Delete a backup of the H2 database from a location
+     *
+     * @param storageLocation The location of the backup file.
+     * @param backupFileName  The name of the backup file.
+     * @return {@code true} if the deletion was successful; {@code false} if any error occurs.
+     */
+    public boolean deleteBackupFileFromStorage(String storageLocation, String backupFileName) {
+        try {
+            // Ensure that storageLocation and backupFileName are not blanks
+            boolean validation = commonsParametersValidation(storageLocation, backupFileName);
+            if (!validation) {
+                return false;
+            }
+            // Ensure that storageLocation ends with a slash
+            storageLocation = normalizePathWithSeparator(storageLocation);
+            String fullBackupFileName = storageLocation + backupFileName;
+
+            final File backupFile = new File(fullBackupFileName);
+            final String backupFilePath = backupFile.getCanonicalPath();
+            // Ensure that storageLocation correspond to an authorised area
+            if (!backupFilePath.startsWith(adminStorageLocation)) {
+                throw new IOException("Backup file is outside of storage file system");
+            } else if (!backupFile.exists()) {
+                throw new IOException("Backup file does not exist");
+            }
+            Path fileToDeletePath = Paths.get(fullBackupFileName);
+            log.info("Starting the delete process [fullBackupFileName:{}]", fullBackupFileName);
+            final long start = System.currentTimeMillis();
+            Files.delete(fileToDeletePath);
+            final long stop = System.currentTimeMillis();
+            log.info("Successfully deleted backup [timestamp:{}, fullBackupFileName:{}, duration:{} ms]", dateFormat.format(new Date(start)), fullBackupFileName, stop - start);
+            return true;
+        } catch (IOException e) {
+            log.error("An error occurred while deleting backup", e);
+        }
+        return false;
+    }
+
+
+    boolean commonsParametersValidation(String storageLocation, String backupFileName) {
```

Comment:
I like this kind of naming for quick `if ... else` reading
```suggestion
    boolean areParametersValid(String storageLocation, String backupFileName) {
```
```suggestion
    boolean checkCommonParameters(String storageLocation, String backupFileName) {
```

Reply:
ok for checkCommonParameters